.github/workflows/test-e2e.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 The Ground Authors.
```

Comment:
In the Android repo we only have one author at the moment, so this would just be "Google LLC" for now. Same throughout.

Reply:
Ah, gotcha. I must have copied it over from the ground-platform. Done.